.github/actions/build/action.yml

```diff
@@ -35,5 +35,7 @@ runs:
         run: |
           export RUST_BACKTRACE=1
           cargo build -p kani-compiler
+          cargo build -p cargo-kani
+          cargo build -p kani-link-restrictions
```

Comment:
should we not just `cargo build`?

Reply:
Yes, I thought we wanted to avoid building everything here for some reason, but that doesn't seem to be the case.